sdk/eventhub/Azure.Messaging.EventHubs/perf/Scenarios/ReadFromPartitionWithReceiver.cs

```diff
@@ -41,7 +41,7 @@ public ReadFromPartitionWithReceiver(SizeCountOptions options) : base(options)
         ///
         public async override Task SetupAsync()
         {
-            await base.SetupAsync();
+            await base.SetupAsync().ConfigureAwait(false);
```

Comment:
Orthogonal but I just realized that I had missed these...

Reply:
I'm surprised the analyzer didn't flag this.

Reply:
The performance test projects flag as test support types, so the analyzer doesn't enforce.   There's no clear guidance on "do or don't" configure awaits in this context; we have people with different opinions.  Even though we're not doing anything with custom sync contexts, I still like to use them, personally.